core/src/main/resources/hudson/slaves/JNLPLauncher/main.jelly

```diff
@@ -37,14 +37,16 @@ THE SOFTWARE.
           ${%Connect agent to Jenkins one of these ways:}
         </p>
         <ul>
-          <li>
-            <p>
-              <a href="slave-agent.jnlp" id="jnlp-link">
-                <img src="${imagesURL}/webstart.gif" alt="${%launch agent}" />
-              </a>
-              ${%Launch agent from browser}
-            </p>
-          </li>
+          <j:if test="${it.launcher.javaWebStartSupported}">
```

Comment:
I know it's jelly, but could we incorporate a link to a resource (wiki, blog, etc) to say why the web starter is not available. So having a `<j:choose>` rather than a `<j:if>` so we can have an else?

Reply:
I'll need help from someone else to make those jelly changes.  I'm not skilled with jelly and need to spend my weekends and nights working on the git plugin and git client plugin.

Reply:
I think it's 

```
<j:choose>
  <j:when test="${it.launcher.javaWebStartSupported}">
[...]
  </j:when>
  <j:otherwise>
    <div>There is no Web starter for the JVM version used to run Jenkins</div>
  </j:otherwise>
</j:choose>
```

Reply:
Same as https://github.com/jenkinsci/jenkins/pull/3766/files#r234416298 .
I believe we need to have a Wiki or jenkins.io page to list known Jenkins core incompatibilities. 
There is https://jenkins.io/doc/administration/requirements/java/ , but probably it worth having a separate page.

Reply:
Let's just add a redirect page on Jenkins.io? This way we can modify the final destination later?

Like https://jenkins.io/redirect/java-web-start ? Or a more general https://jenkins.io/redirect/java-11 ?

Reply:
Let's do `https://jenkins.io/redirect/java11-java-web-start ` or so. It can be finally linking to the anchor on a page, but it can just link to the JIRA ticket for now

Reply:
I agree, +1 on redirect and it can be changed to wiki page in the future. @MarkEWaite do you want me to test / commit the jelly code I provided?

Reply:
@alecharp that would be great.

Reply:
I added the conditional text and redirect link as suggested.